webapp/src/app/shared/widgets/event-widget/event-editor/editor/event-editor.component.ts

```diff
@@ -33,6 +38,12 @@ export class EventEditorComponent implements OnChanges {
 		EventEditorComponent.globalBase = value;
 	}
 	
+	get wrapText(): boolean {
+		return this.wrapTextOverride === undefined?
```

Comment:
can be simplified to 
```typescript
get wrapText(): boolean {
    return this.wrapTextOverride ?? this.sharedService.getWrapEventEditorLinesSetting();
}
```

Also try to avoid calling functions in the template because they get called a billion times (every time angular runs change detection).
A better option would be to calculate the value in `ngOnInit` and cache it or if the function has an input parameter use a pipe instead.